tests/topotests/bgp-path-attributes-topo1/test_bgp_path_attributes.py

```diff
@@ -254,21 +286,35 @@ def test_next_hop_attribute(request):
                                 }
                             }
                         }
+                    },
+                    "ipv6": {
+                        "unicast": {
+                            "neighbor": {
+                                "r1": {
+                                    "dest_link": {
+                                        "r3": {"next_hop_self": True}
+                                    }
+                                }
+                            }
+                        }
                     }
                 }
             }
         }
     }
+
     result = create_router_bgp(tgen, topo, input_dict_1)
     assert result is True, "Testcase {} : Failed \n Error: {}".format(
         tc_name, result)
 
     # Verifying RIB routes
     dut = "r1"
     protocol = "bgp"
-    result = verify_rib(tgen, "ipv4", dut, input_dict, protocol=protocol)
-    assert result is True, "Testcase {} : Failed \n Error: {}".format(
-        tc_name, result)
+    for addr_type in ADDR_TYPES:
+        result = verify_rib(tgen, addr_type, dut, input_dict,
+                            protocol=protocol)
+        assert result is True, "Testcase {} : Failed \n Error: {}".format(
```

Comment:
I think everywhere addr_type would be useful to have in error message, like you did above :+1:

Reply:
If we are asserting for "True" then framework API will take care of printing addresses in info/error messages. 

Above scenario is asserting for "not True" so we are printing error message from script. So I modified.